### ISSUE TITLE
windows react not working?

### ISSUE THREAD

**Issue by Deadvait** [Monika-After-Story/MonikaModDev#8396]
I turned on notifications and windows reacts but none show up when I open Youtube or even this page. I went to the notification settings but DDLC wasn't even showing up in the "Get notifications from these senders", what can I do to fix this?

**Comment by dreamscached** [comment]
What OS do you run the game on?

**Comment by Kzitold** [comment]
I'm also having issues with window reacts.
I have Window Reacts enabled, and all alerts enabled.

**Comment by multimokia** [comment]
It's a little finniky. Sometimes you might need to wait until MAS _tries_ to show a notification for it to show up.

When using MAS and she brings a new topic up, do you ever hear a short piano riff? (Intro to Your Reality)
If so that means Monika _is_ actually trying to show a notification which is a start to see where this is starting to go wrong

**Comment by Pill-9** [comment]
Yes. Not only DDLC or MAS, but all games are not displayed in "get notifications from these senders".I'm using Windows 10.
